skl2onnx/_parse.py

```diff
@@ -271,6 +274,14 @@ def _parse_sklearn_column_transformer(scope, model, inputs,
         return transformed_result_names
 
 
+def _parse_sklearn_grid_search_cv(scope, model, inputs, custom_parsers=None):
+    return (_parse_sklearn_classifier(
+                scope, model, inputs, custom_parsers=None)
+            if type(model.best_estimator_) in sklearn_classifier_list else
```

Comment:
Maybe use this function: https://scikit-learn.org/stable/modules/generated/sklearn.base.is_classifier.html ?

Reply:
I didn't know about this function, looks perfect in this scenario, thanks!